src/plugins/aws/ssm/index.ts

```diff
@@ -280,19 +327,20 @@ const executeScpCommand = async (
   privateKey: string
 ) => {
   // Execute should not leave any ssh-agent processes running after it's done performing an SCP transaction.
-  const execute = `
-  eval $(ssh-agent) >/dev/null 2>&1
-  trap 'kill $SSH_AGENT_PID' EXIT
-  ssh-add -q - <<< '${privateKey}' 
-  ${command}
-  SCP_EXIT_CODE=$? 
-  exit $SCP_EXIT_CODE
-  `;
-
-  return spawnSsmNode({
+  const commands = [
+    // completely disable bash history for this session
+    `unset HISTFILE`,
```

Comment:
This _might_ be overkill because we're spawning a subprocess that will run the commands for us but just in case somehow someone enters that subprocess we're also disabling the history.